book/README.md

```diff
@@ -18,6 +18,21 @@ To install jupyter-book etc.
 cd book/website
 pip install -r requirements.txt
 ```
+[optional] Consider using a virtual environment to isolate the python dependencies from your system python e.g.
+via (replace with python interpreter of your choice)
+```
+cd book/website
+virtualenv -p /usr/bin/python3.7 the-turing-way
+source the-turing-way/bin/activate
+pip install -r requirements.txt
+```
+
+You will also need to make sure that 'bundler' and jekyll are available on your path. I.e., on a vanilla Ubuntu 19.04
```

Comment:
Yes this should be in there for sure. I think the easiest cross-platform solution is to ask people to install ruby (and link out to the ruby installation guidlines) and then `cd book/website; make install` which runs `gem install bundler; bundle install`. What do you think?

Reply:
done